src/components/Dashboard/MainContainer/MainContainer.jsx

```diff
@@ -34,12 +35,31 @@ export default class MainContainer extends Component {
         });
       }
       this.setState(({
-        // dashboardId: id,
+        dashboardId: id,
         taskList: newState,
       }));
     });
   }
 
+  addNewTask = (inputData) => {
+    if (inputData.length > 0) {
```

Comment:
to avoid wrapping all block into if construction you can use
`if (!inputData.length) return`
also in case there will be no input data passed to method - you'll get error "length of undefined", please set default value